WebRoot/css/zm.css

```diff
@@ -1985,123 +1985,193 @@ div.CompDragArea * {
+.ZmContactHeaderInfo .ZmContactHeaderTags:not(:empty) {
+	@ContactHeaderTags@
 }
 
-.contactGroupTableImage {
-	vertical-align:top;
-	text-align:center;
-	padding:1em 5px 0em;
-	height:70px;
+.ZmContactHeaderRow .ZmContactHeaderLocation {
+	@ContactHeaderLocation@
 }
 
-.contactGroupTableImage IMG {
-	@ContactGroupImageRadius@
+.ZmContactHeaderLocation > TABLE {
+	@ContactHeaderLocationTable@
 }
 
-.contactGroupTableContent {
-	padding-right:.5em;
+.ZmContactHeaderLocation .ZmContactHeaderLocationIcon {
+	@ContactHeaderLocationIcon@
 }
 
-.contactGroupTableContent SPAN {
-	@NoWrap@
+.ZmContactHeaderLocation .ZmContactHeaderLocationName {
+	@ContactHeaderLocationName@
+}
+
+.ZmContactHeaderLocation .ContactHeaderLocationLabel {
+	@ContactHeaderLocationLabel@
 }
 
-.contactGroupTableContent .ImgPlus {
+/* Contact Info View Style */
+.ZmContactInfoView .ZmContactInfo:not(:empty) {
+	@ContactInfo@
+}
+
+.ZmContactInfo > TABLE {
+	@FullWidth@
+}
+
+.ZmContactInfo .ZmContactInfoMsg {
+	@ContactInfoMsg@
+}
+
+.ZmContactInfo .ZmContactInfoAction {
+	@ContactInfoAction@
+}
+
+/* Contact view info style */
+.ZmContactInfoView .ZmContactContent {
+	@ContactContent@
+}
+
+.ZmContactContent > TABLE {
+	@FullWidth@
+}
+
+.ZmContactContent .ZmContactRowWrapper {
+	@ContactRowWrapper@
+}
+
+.ZmContactRowWrapper .ZmContactRowLabel {
+	@ContactRowWLabel@
+}
+
+.ZmContactRowWrapper .ZmContactRowValueWrapper {
+	@ContactRowValueWrapper@
+}
+
+.ZmContactRowValueWrapper .ZmContacRowValue {
+	@ContactRowValue@
+}
+
+.ZmContactRowValueWrapper .ZmContacRowValueType {
+	@ContactRowValueType@
+}
+
+/* Contact Group View */
+.ZmContactInfoView .ZmContactGroupListContent {
+	@ContactGroupContent@
+}
+
+.ZmContactGroupItemTable {
+	@ContactGroupItemTable@
+}
+
+.ZmContactGroupItemTable TD {
+	@ContactGroupItemTableCell@
+}
+
+.ZmContactGroupItemImage IMG,
+.ZmContactGroupItemImage .svg-icon {
+	@ContactGroupItemImage@
+}
+
+.ZmContactGroupItemTable .ZmContactGroupItemContent * {
+	@ContactGroupItemContent@
+}
+
+.ZmContactInfoView .ZmContactGroupListContent .ZmContactGroupItemTable .ZmContactGroupItemContent {
+	@ContactGroupItemViewContentWrapper@
+}
+
+.ZmContactInfoView .ZmContactGroupListContent .ZmContactGroupItemTable .ZmContactGroupItemImage {
+	@ContactGroupItemViewImageWrapper@
+}
+
+.ZmContactView .contactHeaderRow,
+.ZmTaskEditView .contactHeaderRow {
+	height:40px;
+	@WidgetHeaderSize-big@
+}
+
+.ZmTaskEditView .ZHasDropDown {
 	display:inline-block;
-	margin-left:.5em;
 }
 
-.ZmContactInfoView .contactHeaderSection {
-	@WidgetHeaderContainer@
-	@WidgetHeaderContainer-normal@
-	@FontSize-big@
-	padding:4px 10px;
-	font-weight:bold;
-	margin-top:10px;
+.newTaskBanner {
+	@InactiveCursor@
+	color:rgb(120, 111, 91);
+	padding:5px 0px;
 }
 
-.ZmContactInfoView .workName {
-	font-weight:bold;
+.newTaskBannerSep {
+	border-bottom:2px solid #AAA;
+}
+
+.ZmEditContactView .contactHeaderRow {
+	@PageBg@
+}
+
+.ZmContactView .contactContentTable {
+	width: 100%;
```

Comment:
This part haven't touched now. I think it's showing as newly added, is because I have removed `.ZmContactInfoView . contactContentTable ` and put all contact view related declarations together, which shifts this declarations down.